llpc/test/shaderdb/relocatable_shaders/TriangleVs_CheckNoteSectionForCacheHash.spvasm

```diff
@@ -0,0 +1,80 @@
+
+; BEGIN_SHADERTEST
+; RUN: amdllpc -spvgen-dir=%spvgendir% -gfxip=9 -unlinked -enable-relocatable-shader-elf -v -add-hash-to-elf %s | FileCheck --check-prefixes=CHECK %s
+; SHADERTEST-LABEL: {{^// LLPC}} SPIRV-to-LLVM translation results
+; SHADERTEST: AMDLLPC SUCCESS
+; END_SHADERTEST
+
+; CHECK: .note (size = 80 bytes)
+; CHECK:     Unknown(0)                (name = llpc_cache_hash  size = 16)
+; CHECK:           0:A798528E BF01F731 27C64DC9 F195ACD3
+; CHECK:     Unknown(0)                (name = llpc_version  size = 8)
+; CHECK:           0:0000002D 00000004
```

Comment:
Would it be possible to combine this and the previous test into a single one with two RUN lines?

Reply:
done